Berksfile

```diff
@@ -12,5 +12,4 @@ if RUBY_VERSION.to_f < 2.0
   cookbook 'ohai', '< 4.0'
 end
 
-cookbook 'hadoop', git: 'https://github.com/caskdata/hadoop_cookbook.git', branch: 'master'
```

Comment:
It was added long ago, for that reason. We needed something that wasn't released. I'm fine with keeping this, but prefer to test using release materials, where possible.